projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -141,11 +169,17 @@ public function __construct( $attributes, $content = null, $form = null ) {
 		if ( ! empty( $attributes['options'] ) && is_string( $attributes['options'] ) ) {
 			$attributes['options'] = array_map( 'trim', explode( ',', $attributes['options'] ) );
 
+			// TODO: Work out where these values are set? Are they actually set anywhere?
```

Comment:
Could it be set by using a shortcode like `[contact-field type="checkbox-multiple" values="1,2,3" /]`?

Perhaps that was an old version of the shortcode.

I see there's also some code here to construct a shortcode with a values attribute like that:
https://github.com/Automattic/jetpack/blob/27ff4357597a0b385570fd0974ded07c27eafe2f/projects/packages/forms/src/contact-form/class-contact-form.php#L879-L884

Reply:
Yeah, I came to the same conclusion it was for BC purposes. Thanks for looking into it.